user_stories.md

```diff
@@ -0,0 +1,11 @@
+# Пользовательские истории.
+
+1. Как пользователь, я хочу иметь возможность создавать сокращенные ссылки без регистрации на сервисе (возможно, с ограниченным временем работы ссылок), чтобы их было удобнее использовать в текстах, так как они[ccылки] будут со значительно меньшим числом символов.
+2. Как пользователь, я хочу переходить по сокращенной ссылке на страницу, которую указывал создатель этой ссылки.
+3. Как пользователь, я хочу узнать, куда ведет сокращенная ссылка, не переходя по ней явно, чтобы обезопасить себя от возможных вирусов.
+4. Как пользователь, я хочу иметь возможность зарегистрироваться и получить свой аккаунт с уникальным id, чтобы меня можно было отличить от других пользователей.
+5. Как зарегистрированный пользователь, я хочу иметь возможность создавать ссылки с произвольным временем работы, и в любой момент иметь возможность удалить их из системы.
+6. Как зарегистрированный пользователь, я хочу иметь возможность задавать ограничения на символы, используемые в тексте сокращенной ссылки, чтобы удовлетворить требованиям контекста использования.
```

Comment:
> Does user create a short link or service?
How do this constraints should look like? Would user specify link's alphabet?

Reply:
If user doesn’t specify concrete short link name, service will genereate it instead. We assume that it can be implemented using the form with checkboxes such as "Do not use Latin", "Do not use capital letters".